targetplatform/JBTIS Oxygen Linux x86_64.launch

```diff
@@ -11,18 +11,18 @@
 <booleanAttribute key="clearwslog" value="false"/>
 <stringAttribute key="configLocation" value="${workspace_loc}/.metadata/.plugins/org.eclipse.pde.core/JBTIS Oxygen Linux x86_64"/>
 <booleanAttribute key="default" value="false"/>
-<stringAttribute key="deselected_workspace_plugins" value="org.fusesource.ide.camel.editor.tests.integration,org.fusesource.ide.camel.model.tests,org.fusesource.ide.camel.validation.tests.integration,org.fusesource.ide.jmx.camel.tests.integration,org.fusesource.ide.launcher.tests.integration,org.fusesource.ide.launcher.ui.tests.integration,org.fusesource.ide.project.tests.integration,org.fusesource.ide.projecttemplates.tests.integration,org.fusesource.ide.server.tests,org.jboss.tools.fuse.transformation.core.tests.integration,org.jboss.tools.fuse.transformation.editor.tests.integration"/>
+<stringAttribute key="deselected_workspace_plugins" value="org.fusesource.ide.camel.editor.tests.integration,org.fusesource.ide.camel.validation.tests.integration,org.fusesource.ide.jmx.camel.tests.integration,org.fusesource.ide.launcher.tests.integration,org.fusesource.ide.launcher.ui.tests.integration,org.fusesource.ide.project.tests.integration,org.fusesource.ide.projecttemplates.tests.integration,org.fusesource.ide.server.tests,org.jboss.tools.fuse.transformation.core.tests.integration,org.jboss.tools.fuse.transformation.editor.tests.integration"/>
 <booleanAttribute key="includeOptional" value="true"/>
 <stringAttribute key="location" value="./ws_fusetooling_dev"/>
 <booleanAttribute key="org.eclipse.jdt.launching.ATTR_USE_START_ON_FIRST_THREAD" value="true"/>
-<stringAttribute key="org.eclipse.jdt.launching.JRE_CONTAINER" value="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/java-8-openjdk"/>
+<stringAttribute key="org.eclipse.jdt.launching.JRE_CONTAINER" value="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/java-8-openjdk-amd64"/>
```

Comment:
I am the only one who launches the tooling on Linux and my path is like this. So I had to change it.